apps/base/utils.py

```diff
@@ -69,3 +74,23 @@ def decode_data(data):
     for i in data:
         decoded.append(base64.decodestring(i+"=="))
     return decoded
+
+
+def send_slack_notification(webhook=settings.SLACK_WEBHOOKS['default'], message=""):
+    '''Send slack notification to any workspace
+
+    Keyword Arguments:
+        webhook {string} -- slack webhook URL (default: {settings.SLACK_WEBHOOKS['default']})
+        message {str} -- JSON/Text message to be sent to slack (default: {""})
+    '''
+    if settings.DEBUG is False:
```

Comment:
You don't need this check here. Please remove it. In dev also, we might want to test the slack notifications sometimes and then this condition will prohibit us from testing that.